tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "declaration": true,
     "rootDir": "src",
     "outDir": "dist",
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "skipLibCheck": true
```

Comment:
This project currently has compilation errors because its tsconfig targets `ES3`, and the type definitions for zod require ES6:
```

> fastify-type-provider-zod@1.1.9 build
> tsc

node_modules/zod/lib/types.d.ts:777:5 - error TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.

777     #private;
        ~~~~~~~~

node_modules/zod/lib/types.d.ts:796:5 - error TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.

796     #private;
        ~~~~~~~~
```

`skipLibCheck` gets rid of them by disabling type-checking `.d.ts` files.
A better approach would be to overhaul the entire config, but that's beyond the scope of this fix.